Sources/NIO/CircularBuffer.swift

```diff
@@ -204,6 +204,13 @@ public struct CircularBuffer<E>: CustomStringConvertible, AppendableCollection {
         return nextIndex
     }
 
+    /// Removes all members from the circular buffer whist keeping the capacity.
+    public mutating func removeAll() {
```

Comment:
In swift in other places we have `removeAll(keepingCapacity:)`. Maybe we should use the same method naming and behaviour ?

https://developer.apple.com/documentation/swift/array/1641473-removeall

Reply:
will do tomo